### DISCUSSION TITLE
記事ページにはてなブックマークのボタンがないのはなぜ？

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#177] [要望・アイデア・不具合]
はてなブックマークは利用者にエンジニアが多いのかホットエントリーに技術記事があがることが多く、Qiitaとの相性がよさそうだと思っています。
ところが、記事ページにTwitterやFacebookのシェアボタンはあるものの、はてなブックマークのボタンはありません（以前はあったようですが）。
これはサービスとしてのポリシーによるものなのでしょうか？

**Comment by kskwtnk** [comment]
@munierujp 
ご意見ありがとうございます！

仰る通り、以前のQiitaにははてなブックマークのボタンがありました
また、Pocketなどもありました

2年半くらい前に以下のような話をしており、整理する運びになりました

- シェアボタンがたくさんありすぎて煩雑な見た目になっている
- たくさん置いてあるものの、実際に使われているのは一部である

現状のQiitaは割と「情報を載せてはいるものの、必要なものが整理できていない」ような状況です
サービスのポリシーとまで言うと少し大仰ですが、ここ最近は「あまり使われないものは落として必要なものにフォーカスする」のを重視しています
そのため、直近でははてなブックマークのボタンを追加する予定はありませんでした

ただ、ボタンをなくす判断をしたのも割と昔ですし、現状のみなさんの意見を聞いてみたい、とも思っています！

**Reply by hanaseleb** [reply]
私はLinkedin のボタンもほしいなぁと思っていました！

**Reply by munierujp** [reply]
特にアナウンスはないようですが、もしかしてはてなブックマークのボタンが追加されてます……？

**Reply by tanaka-asami415** [reply]
@munierujp 
お知らせが遅れてしまい申し訳ありません🙇
こちらのリリースと同時にシェアもしていただけるようになりました！✨
[はてなIDを連携できるようになりました](https://qiita.com/release-notes#%E3%81%AF%E3%81%A6%E3%81%AAid%E3%82%92%E9%80%A3%E6%90%BA%E3%81%A7%E3%81%8D%E3%82%8B%E3%82%88%E3%81%86%E3%81%AB%E3%81%AA%E3%82%8A%E3%81%BE%E3%81%97%E3%81%9F)
リリースノートも追って記載予定です🙏
